web/cypress/integration/upstream/create_and_delete_upstream.spec.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Delete Upstream', () => {
+        const root_name = `root_${new Date().valueOf()}`;
```

Comment:
what is `root_name`?

Reply:
please check the first e2e example, it is written like this, since it was approved, I think the following e2e can be written like this.

Reply:
This `name` is used to define a unique mark for identifying the `Upstream` below.

Reply:
@idbeta your answer is not enough. 

why we need a unique mark for this case?

Reply:
Just for the case of `delete the upstream` to locate this `Upstream` more convenient, if it is not unique, the operation will be complicated.